test/elm/clinical/clinical-test.ts

```diff
@@ -157,6 +161,35 @@ describe('InValueSet', () => {
   });
 });
 
+describe('ExpandValueset', () => {
+  beforeEach(function () {
+    setup(this, data, [], vsets);
+  });
+
+  it('expand valueset', async function () {
+    const received: any = await this.expandFemale.exec(this.ctx);
+    const expected: Code[] = [new Code('F', '2.16.840.1.113883.18.2', 'HL7V2.5', undefined)];
+    received.should.eql(expected);
+  });
+
+  it('expand null', async function () {
+    this.expandNull
+      .exec(this.ctx)
+      .should.be.rejectedWith('ValueSet must be provided to ExpandValueSet function');
```

Comment:
This should expect a `null` result as defined in the CQL specification. https://cql.hl7.org/04-logicalspecification.html#expandvalueset and https://cql.hl7.org/09-b-cqlreference.html#expandvalueset